Makefile

```diff
@@ -70,5 +71,12 @@ run-mypy: ## Run MyPy in Container
 run-local-lineage-server: ## Run flask based local Lineage server
 	FLASK_APP=dev/local_flask_lineage_server.py flask run --host 0.0.0.0 --port 5050
 
+test-rc-dependencies: ## Tests providers RC by building an image with given dependencies and running the master DAG
+	python3 dev/scripts/replace_dependencies.py '$(RC_PROVIDER_PACKAGES)'
+	cd ".circleci/integration-tests/" && \
+	 bash script.sh 'astro-cloud' '$(DOCKER_REGISTRY)' '$(ORGANIZATION_ID)' '$(DEPLOYMENT_ID)' '$(ASTRONOMER_KEY_ID)' '$(ASTRONOMER_KEY_SECRET)'
+	python3 dev/scripts/trigger_master_dag.py '$(DEPLOYMENT_ID)' '$(ASTRONOMER_KEY_ID)' ' $(ASTRONOMER_KEY_SECRET)'
+	git checkout setup.cfg
```

Comment:
The replace_dependencies.py updates the setup.cfg to use the given provider dependencies. The checkout at end removes the changes and restores the original file so that those do not get checked in by mistake.